### DISCUSSION TITLE
May I ask how to split the layers in a TIF file and export them like Photoshop, without cutting the layers and keeping the layers in the background position

### DISCUSSION THREAD

**Discussion by 291598912** [ImageMagick/ImageMagick#6306] [Digital Image Processing]
May I ask how to split the layers in a TIF file and export them like Photoshop, without cutting the layers and keeping the layers in the background position

**Comment by snibgo** [comment]
I don't use Photoshop. What does "export" mean? This simple command:
```
magick in.tiff out.png
```
... will write one output PNG file per input image. Does that do what you want?

**Reply by 291598912** [reply]
The difference between these two images is that one is cropped. One is uncut

**Reply by 291598912** [reply]
![im_zimu](https://user-images.githubusercontent.com/16145102/236775195-e6f9748f-9d13-4623-ad2a-03dab0ae355f.png)
![边框字母左2](https://user-images.githubusercontent.com/16145102/236775211-0f3f7bc4-4d73-4b20-98cc-f2bc12143376.png)

**Comment by 291598912** [comment]
[intput.zip](https://github.com/ImageMagick/ImageMagick/files/11419283/intput.zip)
This is a tif file
![im_zimu](https://user-images.githubusercontent.com/16145102/236774621-bf2420fd-bc98-4196-92c4-0328a048f19a.png)
![边框字母左2](https://user-images.githubusercontent.com/16145102/236774677-1ebfc911-b7d9-43e4-b0cc-c723e6fdb720.png)

The difference between these two images is that one is cropped. One is uncut

**Reply by 291598912** [reply]
This command:
magick intput.tif -profile sRGB.icc %d.png

**Reply by 291598912** [reply]
![8](https://user-images.githubusercontent.com/16145102/236781958-d60b0652-bf47-495c-b456-17fd7e664069.png)
![镜头框](https://user-images.githubusercontent.com/16145102/236781987-b3e2248c-722f-41af-95b1-b42eb547f090.png)
What I said about not cropping is the difference between these two images

**Reply by 291598912** [reply]
![8](https://user-images.githubusercontent.com/16145102/236782539-206179b8-afe8-4942-a214-21b2d2397b6b.png)

**Reply by 291598912** [reply]
Is there a solution to this problem

**Reply by 291598912** [reply]
I want a large image, not the generated small image
